### PR TITLE
chore: release v1.0.0-dev.1

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

**Comment by Vinzent03** [comment]
May you add the changes of #29 to the new version and release it ?

**Comment by Vinzent03** [comment]
@dshukertjr  May you add the changes of #29 to the new version and release it ?

**Comment by dshukertjr** [comment]
@Vinzent03 
Sounds good!

**Review by bdlukaa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [0.1.0]
+
+- BREAKING: error is now thrown instead of returned within the responses.
```
[the remark this answers is not in the document]
Comment (dshukertjr) [reply CHANGELOG.md:3]: @bdlukaa Amazing idea! Adding it now!

**Review by bdlukaa — APPROVED** [review]

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**